src/__tests__/fixtures/json-schema/orders.json

```diff
@@ -38,7 +40,9 @@
 						"type": "integer"
 					},
 					"price": {
-						"type": "number"
+						"type": "number",
+						"sort": true,
```

Comment:
Sort/Faceting is not supported for arrays, should we through a client side array in this case?

Reply:
As discussed. Will handle it in a separate change.